### PR TITLE
Statements pdf or html and language

### PR DESCRIPTION
**Pull request by axshani**
small changes to statements
added language filter

### CODE REVIEW

File: resources/statements.ts [hunk 1]
```diff
@@ -59,3 +59,18 @@ export interface StatementsListParams {
...
 }
+
+
+export class GetStatementRequest {
+    public statementId: string
+    public outputType: string
+    public customerId: string | undefined
+    public language: string
+
+    constructor(statementId: string, outputType: string = "html", language: string = "en", customerId?: string) {
```
Comment (unit-shahaf) [inline resources/statements.ts:70]: Please change `outputType` and `language` form string to new types containing their possible values.
especially because we are using  `outputType` to determine to which route to send the request

File: tests/statements.spec.ts [hunk 2]
```diff
@@ -18,7 +19,7 @@ describe("Statements List", () => {
 
 describe("Get Statement Test", () => {
     test("get one statement", async () => {
-        const res = await unit.statements.get(statementId[0])
+        const res = await (await unit.statements.get(new GetStatementRequest(statementId[0]))).data
```
Comment (unit-shahaf) [inline tests/statements.spec.ts:22]: Why did you add another  `await` here?

**Review by unit-shahaf — CHANGES_REQUESTED** [review]
This is a breaking change we will need to increase the minor version after merging this PR

**Comment by axshani** [comment]
thanks, I made the requested changes

**Review by unit-shahaf — APPROVED** [review]

**Comment by axshani** [comment]
https://github.com/unit-finance/unit-node-sdk/issues/154

added responseType to get request
